docs/guides/missionmaking.md

```diff
@@ -112,6 +112,14 @@ _Jeżeli zamierzasz bawić się skryptami to bardzo polecamy `Visual Studio Code
 
 W naszych misjach pojawiają się różne treści. Takie rzeczy można przygotowywać w pliku stringtable.xml. Misja pokazowa ma tam zdefiniowaną nazwę nazwę misji oraz treść intelu. Najprościej przyjąć, że to tablica stałych tekstowych.
 
+**UWAGA**
+Nie pomijaj tego punktu. Jeśli nie masz potrzeby niczego zmieniać przynajmniej edytuj nazwę misji. W miejsce `ArmaForces - Preset` wpisz swoją unikalną nazwę misji. Unikniesz dzięki temu problemów z dublowaniem się nazw na serwerze i tym samym brakiem Twojej misji na liście. Najlepiej wyeduj wszystkie pola, oprócz korzyści czysto porządkowych gracze zobaczą większe napracowanko. 
+```xml
+<Key ID="STR_AFP_Scripts_DisplayName">
+	<Polish>ArmaForces - Preset</Polish>
+</Key>
+```
+
```

Comment:
I think it could be nice to elaborate a bit on how to create new stringtable keys.

Each key should contain `STR_AFP_Scripts` part unchanged, otherwise it won't work correctly (or at all). And mention that language highest on the list is the default one if correct language is not available.

If the configuration would be like below, then someone with German language would see Polish strings (which is nonsense most of the time).
```xml
<Key ID="STR_AFP_Scripts_SomeString">
    <Polish>Jakiś string</Polish>
    <English>Some string</English>
</Key>
```